### ISSUE TITLE
Feature Request: Embeddable UI for specific jobs

### ISSUE THREAD

**Issue by Shelob9** [reaktivstudios/locomotive#118]
I'm using this for a [developer-focused plugin to help me/ others manage EDD}(https://github.com/CalderaWP/cwp-bundle-payment-updates), but I'm also evaluating it for use for bulk CSV exports in Caldera Forms. For my current use, I don't really care that users (me/ whoever finds my plugin on Github) will have to go to tools->batches to run this, but in a user-facing plugin, that's a bad UX.

It would be really cool if I have a way to embed this UI, which I love beacuse it's good and I didn't have to build it, in my plugin's admin screen.

**Comment by Mte90** [comment]
I agree this plugin will be interesting if can be embedded in others using a method that print the ui as example with the custom batch list.
Else we need to add locomotive as dependence or package inside the plugin itself and is not a good approach if we cannot create a custom ui.
